docs/biosample.rst

```diff
@@ -71,11 +71,11 @@ genomic array as well as RNA-seq experiments) may refer to the same Biosample.
      - required
      - The procedure used to extract the biosample
    * - hts_files
-     - :ref:`rstfile`
+     - :ref:`rstfile` (list)
      - optional
      - list of high-throughput sequencing files derived from the biosample
```

Comment:
Capitalise 'L' in 'list'

Reply:
@julesjacobsen Done. I'll review the other tables as well, I think there are some instances where tables are all lowercase and others are capitalized. I'll make them all capitalized for consistency.